### PR TITLE
Annotations: Fix annotation title

### PR DESCRIPTION
**Pull request by adela-almasan**
- Fixes annotation title
- Updates alert text

**Special notes for your reviewer:**
- duplicated `getAlertAnnotationInfo` function as it's being used by Graph

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/alerting/state/alertDef.ts [hunk 1]
```diff
@@ -207,6 +207,25 @@ function getAlertAnnotationInfo(ah: any) {
   return '';
 }
 
+// Copy of getAlertAnnotationInfo, used in annotation tooltip
+function getAlertAnnotationText(annotationData: any) {
+  // backward compatibility, can be removed in grafana 5.x
```
Comment (codeincarnate) [inline public/app/features/alerting/state/alertDef.ts:212]: Seems like this shouldn't be here?

Comment (baldm0mma) [reply public/app/features/alerting/state/alertDef.ts:212]: I mean, it's true... 😂

**Review by baldm0mma — APPROVED** [review]
LGTM!

File: public/app/features/alerting/state/alertDef.ts (hunk 1 above)
Comment (adela-almasan) [reply public/app/features/alerting/state/alertDef.ts:212]: probably the entire copy function shouldn't be here 😅 but unsure how to deal with its usage in graph rn..

Comment (baldm0mma) [reply public/app/features/alerting/state/alertDef.ts:212]: LOL! I think a little copypasta is great for the first move!

**Pull request merged**